openfecli/commands/plan_rbfe_network.py

```diff
@@ -145,6 +148,9 @@ def plan_rbfe_network(
         folder_path=OUTPUT_DIR.get(output_dir),
     )
 
+    end_time = datetime.now()
```

Comment:
May I suggest using a decorator for this?

Reply:
problem is, thenyou can not align it anymore within the output. but maybe that is the way...

Reply:
Don't we have a utils area?

Reply:
I think I got an idea